tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/ArrayAppendTest.php

```diff
@@ -21,6 +21,8 @@ protected function getExpectedSqlStatements(): array
         return [
             'SELECT array_append(c0_.array1, 1989) AS sclr_0 FROM ContainsArrays c0_',
             "SELECT array_append(c0_.array1, 'country') AS sclr_0 FROM ContainsArrays c0_",
+            "SELECT array_append(c0_.array1, ?) AS sclr_0 FROM ContainsArrays c0_",
+            "SELECT array_append(c0_.array1, 2 * ?) AS sclr_0 FROM ContainsArrays c0_",
```

Comment:
sorry this was an intermediate try, i forgot to push my commit --amend , i will do that in a few hours